docs/development.md

```diff
@@ -69,15 +69,62 @@ npm run pretty
 npm run type-check
 ```
 
-### Summary of Server-Side Rendering and Client-Side Data Handling for Jobs and Chat Routes
+## Local Dev Chat Environment
+
+### 1) Using the ilab command line tool
+
+For the chat functionality to work you need a ilab model chat instance. To run this locally:
+
+`cd server`
+
+[https://github.com/instructlab/instructlab?tab=readme-ov-file#-getting-started](https://github.com/instructlab/instructlab?tab=readme-ov-file#-getting-started)
+
+After you use the `ilab serve` command you should have, by default, a chat server instance running on port 8000.
+
+### 2) Using docker
```

Comment:
Mind changing to Podman because of their Docker desktop licensing we are using Podman. Ty!

Reply:
Done